src/components/inspectors/InspectorPanel.vue

```diff
@@ -106,7 +106,7 @@ export default {
     }, saveDebounce),
     defaultInspectorHandler(value, node, setNodeProp) {
       /* Go through each property and rebind it to our data */
-      for (const key in value) {
+      for (const key in omit(value, ['$type', 'eventDefinitions'])) {
```

Comment:
I think that Array properties (like eventDefinitions) represents child nodes and not attributes, may be this could be improved